src/main/java/com/cosc436002fitzarr/brm/services/UserService.java

```diff
@@ -1,8 +1,13 @@
 package com.cosc436002fitzarr.brm.services;
```

Comment:
You'll need to fix the merge conflicts in this file. On line 29, it looks like you didn't commit some of the changes that were pushed into master branch for the /createUser API.